### PR TITLE
Document cross-platform filesystem requirements

### PR DESCRIPTION
**Pull request by heyitsaamir**
## Summary

Windows release runners motivated documenting the repository's cross-platform filesystem expectations for contributors and agents.

### CODE REVIEW

**Review by corinagum — APPROVED** [review]

File: AGENTS.md [hunk 1]
```diff
@@ -9,3 +9,4 @@
 
 - Tests should be thoughtfully created to exercise the intended purpose, not just to turn green. When possible, do red-green tests: confirm the test fails when the behavior is wrong and passes when it is right, so it actually guards the behavior it claims to.
 - Consider mocking carefully. The goal is to guard against regressions for the intended behavior, so mocks should mirror the real contract (types, shapes, and edge values like `null`) rather than a simplified stand-in that a broken implementation could still satisfy.
+- Release validation may run on Windows as well as Linux. Keep filesystem code and tests cross-platform: use Node and `path` APIs instead of hard-coded separators, specify UTF-8 for text APIs whose encoding is platform-dependent, and avoid locale-dependent or Unix-only filesystem assumptions.
```
[the remark this answers is not in the document]
Comment (heyitsaamir) [reply AGENTS.md]: @copilot Fix the code for all comments in [this review comment](https://github.com/microsoft/teams.ts/pull/691#discussion_r3707376199).

When a review comment includes a suggested change, apply the suggestion exactly.

Do not make changes beyond what is described in the linked review comment.

**Review by MehakBindra — APPROVED** [review]
